pkg/resources/common/formatter_test.go

```diff
@@ -1072,3 +1075,223 @@ func Test_formatterLinks(t *testing.T) {
 		})
 	}
 }
+
+func TestFormatterAddsResourcePermissions(t *testing.T) {
+	const (
+		clusterid = "clusterid"
+		projectid = "projectid"
+	)
+
+	tests := []struct {
+		name                string
+		topLevelPermissions []string
+		resourcePermissions map[string][]string
+		schema              *types.APISchema
+		apiObject           types.APIObject
+		want                map[string]map[string]string
+	}{
+		{
+			name:                "get update patch on project and get on projectroletemplatebindings",
+			topLevelPermissions: []string{"get", "update", "patch"},
+			resourcePermissions: map[string][]string{
+				"projectroletemplatebindings": {"get", "list", "watch"},
```

Comment:
Would it be possible to use camelCase for this PRTB key and make it easier to read?